docs/faq.md

```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
 
 To inspect the image that is produced from the build using Docker, you can use commands such as `docker inspect your/image:tag` to view the image configuration, or you can also download the image using `docker save` to manually inspect the container image. Other tools, such as [dive](https://github.com/wagoodman/dive), provide nicer UI to inspect the image.
 
+### How do I specify an image from a manifest list?
+
+By design, Jib will pick always select an image for the platform:`amd64/linux` if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list). If you need to specify a different image from a manifest list you must specify the image digest for the platform you are targeting. 
+
+To determine the digest of an image in a manifest list, you can view the manifest list with:
+```
+$ docker manifest inspect openjdk:8
```

Comment:
The `manifest inspect` command is an experimental one and needs to update `~/.docker/config.json`. Maybe we can add some links:
https://docs.docker.com/engine/reference/commandline/manifest_inspect/ https://docs.docker.com/engine/reference/commandline/cli/#experimental-features